src/flamenco/runtime/context/fd_exec_instr_ctx.c

```diff
@@ -16,14 +16,14 @@ fd_instr_borrowed_account_view_idx( fd_exec_instr_ctx_t * ctx,
 
 int
 fd_instr_borrowed_account_modify_idx( fd_exec_instr_ctx_t *    ctx,
-                                      uchar                    idx,
+                                      ulong                    idx,
                                       ulong                    min_data_sz,
                                       fd_borrowed_account_t ** account ) {
   if( FD_UNLIKELY( idx >= ctx->instr->acct_cnt ) )
     return FD_ACC_MGR_ERR_UNKNOWN_ACCOUNT;
   if( FD_UNLIKELY( !fd_instr_acc_is_writable_idx( ctx->instr, idx ) ) ) {
     /* FIXME: we should just handle the try_borrow_account semantics correctly */
-    FD_LOG_DEBUG(( "unwritable account passed to fd_instr_borrowed_account_modify_idx (idx=%u)", idx ));
+    FD_LOG_DEBUG(( "unwritable account passed to fd_instr_borrowed_account_modify_idx (idx=%lu)", idx ));
```

Comment:
I wonder if this should be at least a warning.  In the case of the bug yesterday, it caused a segfault later...